### PR TITLE
LearnOpenTK - Improve tutorials/Add 1.3

### PR DESCRIPTION
**Pull request by devvoid**
List of changes:

Tutorial 0 has been changed from a guide on how to install OpenTK from Nuget to a brief overview of OpenGL, to better match with the original tutorial. Content from the original Tutorial 0 has been merged with Tutorial 1. Tutorial 2 has had all of its images fixed, and has been updated to match the code of the C# version.

### CODE REVIEW

**Comment by devvoid** [comment]
@DylanFPS I think that's it for now, this is ready for review.

**Comment by devvoid** [comment]
Forgot to explain a few of the overloaded functions being used, added that in now.

File: learn/chapter1/3-element-buffer-objects.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Element Buffer Objects
+We've rendered our first triangle, but what if we wanted to do something more complex? For instance, a rectangle?
+
+OpenGL works exclusively in triangles. It has partial support for squares and other shapes, but those have been depreciated for a very long time and should not be used. You could do it by defining six vertices to make two triangles that look like they're one shape, but that would be a waste since a rectangle only needs four vertices. While that might not seem terrible, consider a cube: only eight vertices if none are wasted, but if you used the approach of making each face out of two triangles, that'd take 36 vertices just for a single cube! The numbers only get worse the more complex your model is; clearly, a better way is needed.
```
Comment (DylanFPS) [inline learn/chapter1/3-element-buffer-objects.md]: Wrong word used.

depreciated -> deprecated

File: learn/chapter1/1-creating-a-window.md [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+
+![Tools > NuGet > Package Manager Console](1-dropdown-nuget.png)
+
+This will bring up the Package Manager Console - a PowerShell extension for Visual Studio and NuGet. To install OpenTK, simply type the following command into the Package Manager Console.
+
+```
+Install-Package OpenTK
+```
+
+![Install OpenTK](1-nuget-package-manager.png)
```
[the remark this answers is not in the document]
Comment (devvoid) [reply learn/chapter1/1-creating-a-window.md]: Do you mean the command, or the image?

File: learn/chapter1/3-element-buffer-objects.md (hunk 1 above)
Comment (devvoid) [reply learn/chapter1/3-element-buffer-objects.md]: Fixed

File: learn/chapter1/1-creating-a-window.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (devvoid) [reply learn/chapter1/1-creating-a-window.md]: Nevermind, found the issue, extension on the file was in allcaps for some reason. Fixed.

**Pull request merged**